src/typelib.erl

```diff
@@ -48,7 +48,10 @@ pp_type(Type = {type, _, bounded_fun, _}) ->
     "fun(" ++ S ++ ")";
 pp_type({var, _, TyVar}) ->
     %% TODO: In type(), TyVar should be an atom but we use a string.
-    TyVar;
+    case is_atom(TyVar) of
+        true -> atom_to_list(TyVar);
+        false -> TyVar
+    end;
```

Comment:
Maybe it's a matter of preference, but I prefer guards. This also avoids catch-all. (Better crash on unexpected data.)

```suggestion
    if
        is_atom(TyVar) -> atom_to_list(TyVar);
        is_list(TyVar) -> TyVar
    end;
```

Reply:
Thanks!